CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+Contributing to the google_fonts package.
+=========================================
+
+If you're interested in contributing to the `google_fonts` package, thank you! We love receiving 
```

Comment:
"Thank you for your interest in contributing to the `google_fonts` package!"

Reply:
Done.